src/api/apiClient.ts

```diff
@@ -107,6 +109,12 @@ const getStewardshipActivities = (
   ).then((res) => res.data);
 };
 
+const getAllCommonNames = (): Promise<string[]> => {
+  return AppAxiosInstance.get(
+    ParameterizedApiRoutes.GET_ALL_COMMON_NAMES(),
+  ).then((res) => res.data.names);
```

Comment:
⛏️ Could we have `getAllCommonNames` return just `res.data` here (instead of `res.data.names`) so that it's consistent with the other routes?